common-app-lib/src/cmr/common_app/api/log.clj

```diff
@@ -0,0 +1,133 @@
+(ns cmr.common-app.api.log
```

Comment:
Very nicely done ns/file. 👍

Reply:
Thanks. The comments have been moved to cmr.common-app.services.logging